test/test_helper.rb

```diff
@@ -27,3 +36,10 @@
 module API
   Person = ::Google::Protobuf::DescriptorPool.generated_pool.lookup("pbbuildertest.Person").msgclass
 end
+
+class Racer < Struct.new(:id, :name, :friends, :best_friend)
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+end
```

Comment:
You want to avoid inheriting from `Struct` because it can leave an extra, unused class allocation. Instead, do something like this:
```suggestion
Racer = Struct.new(:id, :name, :friends, :best_friend, keyword_init: true) do
  extend ActiveModel::Naming
  include ActiveModel::Conversion
end
```

I added the `keyword_init` option because that's how you'd typically expect a class to behave when initializing.

Reply:
This is tests, who cares